test/Poseidon/JannoSpec.hs

```diff
@@ -29,55 +31,62 @@ testPoseidonSampleFromJannoFile = describe "Poseidon.Janno.readJannoFile" $ do
     let normalPartialJannoPath    = "test/testDat/testJannoFiles/normal_partial.janno"
     let borkedFullJannoPath       = "test/testDat/testJannoFiles/borked_full.janno"
     let borkedPartialJannoPath    = "test/testDat/testJannoFiles/borked_partial.janno"
-    let borkedWrongNameJannoPath  = "test/testDat/testJannoFiles/borked_wrong_name.janno"
     it "should read minimal janno files correctly" $ do
```

Comment:
Uff, I think there should be a better way than doing these endless tests. I guess one could just enter one datastructure that maps the test file, and then just do one large comparison of this structure against the one loaded...

Reply:
Ja - I'm also not happy with that. We should try to come up with something more elegant

Reply:
#155